### PR TITLE
[Improvement](aggregation) make fixed hashmap's bitmap_size flexable

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes
make fixed hashmap's bitmap_size flexable

```sql
select count(1) from ( select ss_customer_sk customer_sk       ,ss_item_sk item_sk from store_sales,date_dim where ss_sold_date_sk = d_date_sk   and d_month_seq between 1199 and 1199 + 11 and ss_sold_date_sk IS NOT NULL group by ss_customer_sk         ,ss_item_sk ) a;
6s890ms
                              -  GetResultsTime:  419.765ms
                              -  HashTableComputeTime:  5s851ms
                              -  HashTableEmplaceTime:  5s424ms
                              -  HashTableInputCount:  55.285563M  (55285563)
                              -  HashTableIterateTime:  275.381ms
                              -  HashTableSize:  54.116764M  (54116764)
                              -  InsertKeysToColumnTime:  113.262ms
                              -  MaxRowSizeInBytes:  0
                              -  MemoryUsage:  
                                  -  HashTable:  2.50  GB
                                  -  PeakMemoryUsage:  4.13  GB
                                  -  SerializeKeyArena:  1.63  GB

6s741ms
                              -  GetResultsTime:  421.935ms
                              -  HashTableComputeTime:  5s748ms
                              -  HashTableEmplaceTime:  5s337ms
                              -  HashTableInputCount:  55.285553M  (55285553)
                              -  HashTableIterateTime:  246.168ms
                              -  HashTableSize:  54.116764M  (54116764)
                              -  InsertKeysToColumnTime:  106.705ms
                              -  MaxRowSizeInBytes:  0
                              -  MemoryUsage:  
                                  -  HashTable:  2.00  GB
                                  -  PeakMemoryUsage:  2.90  GB
                                  -  SerializeKeyArena:  919.87  MB



```

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 45.15 seconds
 stream load tsv:          545 seconds loaded 74807831229 Bytes, about 130 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          29.0 seconds inserted 10000000 Rows, about 344K ops/s
 storage size: 17162212791 Bytes

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 47.03 seconds
 stream load tsv:          509 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.4 seconds inserted 10000000 Rows, about 340K ops/s
 storage size: 17162186380 Bytes

**Comment by BiteTheDDDDt** [comment]
run p1

**Review by HappenLee — APPROVED** [review]
LGTM

File: be/src/vec/common/uint128.h [hunk 1]
```diff
@@ -192,6 +192,19 @@ struct UInt256 {
...
     }
 };
+
+#pragma pack(1)
+struct UInt136 {
+    UInt8 a;
+    UInt64 b;
+    UInt64 c;
+
+    bool operator==(const UInt136 rhs) const { return a == rhs.a && b == rhs.b && c == rhs.c; }
```
Comment (HappenLee) [inline be/src/vec/common/uint128.h:202]: better can use <=> in c++20 reduce the code
```
    auto operator<=>(const UInt136& v) const = default;
```

**Review by zhangstar333 — APPROVED** [review]

**Pull request merged**